README.md

```diff
@@ -1,3 +1,41 @@
-# rhdm-7-openshift-image
-Red Hat Decision Manager 7 OpenShift images
+# Red Hat Decision Manager 7 OpenShift images
 
+This repository contain all the image descriptors and files necessary to build the RHPAM images.
+Also includes the application templates.
+
+
+### Repo structure:
+rhdm-7-openshift-image \
+├── [controller](controller): RHDM Controller  image descriptor files. \
+├── [decisioncentral](decisioncentral): Decision Central image descriptor files. \
+├── [decisioncentral-indexing](decisioncentral-indexing): Decision Central Indexing (elasticsearch) image descriptor files. \
+├── [kieserver](kieserver): Execution Server image descriptor files. \
+├── [optaweb-employee-rostering](optaweb-employee-rostering): Optaweb Employee Rostering image descriptor files. \
+├── [quickstarts](quickstarts): quickstarts used as example in some of the application templates. \
+└── [templates](templates): Application templates, used to instantiate a new RHDM environment on OpenShift. \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;├── [docs](templates/docs):  Application templates docs generated by the [gen_template_docs.py](https://github.com/jboss-container-images/jboss-kie-modules/blob/master/tools/gen-template-doc/gen_template_docs.py) script. 
+
+Inside each image directory you will find the following files:
+
+ - **container.yaml**: used by OSBS builds.
+ - **content_sets.yaml**: define the yum repositories needed to install dependencies for the image.
+ - **branch-overrides.yaml**: overrides file which uses the latest stable version for external dependencies. Module.yaml and image.yaml always refer to master branch.
```

Comment:
I know it's the beginning of a sentence, so normally capitalized, however the file module.yaml starts with a lower-case 'm'. So perhaps change the sentence to: "The module.yaml and image.yaml files always refer to the master branch."